sdk/client/src/listeners/network-listener/utils/web-socket-listener.ts

```diff
@@ -0,0 +1,20 @@
+import { WebSocketEvent } from '../utils/models'
+
+export type WebSocketListenerCallback = (event: WebSocketEvent) => void
+export type HighlightWebSocketWindow = Window & {
+	_highlightWebSocketCallback: WebSocketListenerCallback
+	WebSocket: any
+}
+
+declare var window: HighlightWebSocketWindow
+
+const WebSocketListener = (callback: WebSocketListenerCallback) => {
+	const initialHighlightWebSocketCallback = window._highlightWebSocketCallback
+	window._highlightWebSocketCallback = callback
```

Comment:
Should we also block urls from the `urlBlocklist` here?

Follow up - what does blocking look like? Do we just not show the message, but that an event happen, or hide the event entirely?